commercetools-models/src/main/java/io/sphere/sdk/products/search/LocaleSelectionBuilder.java

```diff
@@ -0,0 +1,47 @@
+package io.sphere.sdk.products.search;
+
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.models.Builder;
+
+import javax.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public final class LocaleSelectionBuilder extends Base implements Builder<LocaleSelectionDsl> {
+    @Nullable
+    private List <String> localeProjection = new ArrayList<>();
+
+    LocaleSelectionBuilder(@Nullable final List<String> localeProjection) {
+        this.localeProjection = localeProjection;
+    }
+
+    private LocaleSelectionBuilder localeProjection(@Nullable final String localeProjection) {
+        this.localeProjection = new ArrayList<>();
+        this.localeProjection.add(localeProjection);
+        return this;
+    }
+
+    private LocaleSelectionBuilder plusLocaleProjection(@Nullable final String localeProjection) {
+        if (this.localeProjection == null) {
+            this.localeProjection = new ArrayList<>();
+        }
+        this.localeProjection.add(localeProjection);
+        return this;
+    }
+
+    public static LocaleSelectionBuilder of(@Nullable final String localeProjection) {
```

Comment:
please add an additional method which allows the usage of List localeProjections

e.g.: `of(@Nullable final List<String> localeProjection)`